recipes/fastsrm/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "fastsrm" %}
+{% set version = "0.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastsrm-{{ version }}.tar.gz
+  sha256: fe6655b39e3ee15c9ad526819f0832616818042e8589512658087b4b3bb81ee4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.0
+  run:
+    - joblib
+    - matplotlib-base
+    - python >=3.0
+    - python-picard
+    - scikit-learn
+    - tqdm
+    - numpy >=1.12
+    - scikit-learn >=0.23
+    - python-picard
+    - tqdm
+    - pytest
+    - pytest-timeout
+    - pytest-cov
+    - coverage
```

Comment:
Are these really runtime requirements? They look like they would only be useful when running tests

Reply:
Indeed they are only necessary to run the tests I'll remove these